lib/fencing/st_client.c

```diff
@@ -442,17 +442,22 @@ make_args(const char *action, const char *victim, uint32_t victim_nodeid, GHashT
             alias = g_hash_table_lookup(port_map, victim);
         }
 
-        /* Always supply the node's name too:
-         *    https://fedorahosted.org/cluster/wiki/FenceAgentAPI
-         */
-        append_const_arg("nodename", victim, &arg_list);
```

Comment:
You'd have to check "port" for valid contents anyway wouldn't you?
Anyway, how about https://github.com/ClusterLabs/pacemaker/commit/6497493 ?